### DISCUSSION TITLE
What's the future of this library after MDC-based Angular Material v15?

### DISCUSSION THREAD

**Discussion by nunoarruda** [ng-matero/extensions#148] [Q&A]
[Angular Material v15](https://github.com/angular/components/releases/tag/15.0.0) is out which includes a [large refactor](https://blog.angular.io/angular-v15-is-now-available-df7be7f2f4c8#f348) that makes Angular Material components based on [MDC Web](https://m2.material.io/components?platform=web).

Since Angular Material is now tightly coupled with the Material Design spec, the Angular team also had to deprecate/remove some features like, for example, [inverted or vertical sliders](https://github.com/angular/components/blob/main/guides/v15-mdc-migration.md#slider).

What's the future of this library after MDC-based Angular Material v15?

**Comment by nzbin** [comment]
It doesn't matter. Most components don't depend on Angular Material and It will update to v15 as scheduled.
